Compressonator/Applications/_Plugins/CAnalysis/Analysis/CMakeLists.txt

```diff
@@ -35,12 +35,28 @@ set_property(TARGET Analysis PROPERTY POSITION_INDEPENDENT_CODE TRUE)
 find_package(Qt5Gui)
 if(Qt5Gui_FOUND)
 target_include_directories(Analysis
-                           PRIVATE 
+                           PRIVATE
                            "${Qt5Gui_INCLUDE_DIRS}")
 else()
 message(FATAL_ERROR "Package Qt5 (Qt5Gui) are required, but not found. In Unix, run initsetup_unix.sh or sudo apt-get install qtdeclarative5-dev. If is window, please make sure install qt and add the bin path (i.e. C:\Qt\5.7\msvc2015_64\bin) to environment PATH.")
 endif()
 
+find_path(ILMBASE_INCLUDE_DIR OpenEXR/half.h)
+find_path(OPENEXR_INCLUDE_DIR OpenEXR/ImfNamespace.h)
+if(ILMBASE_INCLUDE_DIR)
+    target_include_directories(CompressonatorCLI-bin
```

Comment:
You should not define include directories for the CompressonatorCLI-bin target in a unrelated CMakeLists.txt, but only in Compressonator/Applications/CompressonatorCLI/CMakeLists.txt .